src/Atmos/Model/tendencies_tracers.jl

```diff
@@ -0,0 +1,29 @@
+##### Moisture tendencies
+
+#####
+##### First order fluxes
+#####
+
+function flux(::Advect{Tracers{N}}, m, state, aux, t, ts, direction) where {N}
+    u = state.ρu / state.ρ
+    return (state.tracers.ρχ .* u')'
```

Comment:
Can you teach me what are the `'` doing here?

Reply:
`'` is the adjoint (IIUC, here, this is effectively a transpose). Size-wise, I think we have
```
(state.tracers.ρχ .* u')' = ((N x 1) .* (3 x 1)')' = ((N x 1) .* (1 x 3))' = (N x 3)' = (3 x N) # (3 for 3 component flux, N for N tracers)
```